pom.xml

```diff
@@ -20,7 +20,7 @@
   <properties>
     <java.version>17</java.version>
     <version.spring.cloud>2022.0.0-M4</version.spring.cloud>
-    <version.spring.cloud.azure>6.0.0-beta.1</version.spring.cloud.azure>
+    <version.spring.cloud.azure>6.0.0-beta.2</version.spring.cloud.azure>
```

Comment:
It will have error if some artifacts do not exist in `spring-cloud-dependencies:6.0.0-beta.2`